packages/react-examples/src/react/ScrollablePane/ScrollablePane.DetailsList.Example.tsx

```diff
@@ -140,36 +148,31 @@ export const ScrollablePaneDetailsListExample: React.FunctionComponent = () => {
     setItems(text ? allItems.filter((item: IScrollablePaneDetailsListExampleItem) => hasText(item, text)) : allItems);
   };
   return (
-    <div className={classNames.wrapper}>
-      <ScrollablePane scrollbarVisibility={ScrollbarVisibility.auto}>
-        <Sticky stickyPosition={StickyPositionType.Header}>
-          <TextField
-            className={classNames.filter}
-            label="Filter by name:"
-            // eslint-disable-next-line react/jsx-no-bind
-            onChange={onFilterChange}
-          />
-        </Sticky>
-        <Sticky stickyPosition={StickyPositionType.Header}>
-          <h1 className={classNames.header}>Item list</h1>
-        </Sticky>
-        <MarqueeSelection selection={selection}>
-          <DetailsList
-            items={items}
-            columns={columns}
-            setKey="set"
-            layoutMode={DetailsListLayoutMode.fixedColumns}
-            constrainMode={ConstrainMode.unconstrained}
-            onRenderDetailsHeader={onRenderDetailsHeader}
-            onRenderDetailsFooter={onRenderDetailsFooter}
-            selection={selection}
-            selectionPreservedOnEmptyClick
-            ariaLabelForSelectionColumn="Toggle selection"
-            ariaLabelForSelectAllCheckbox="Toggle selection for all items"
-            onItemInvoked={onItemInvoked}
-          />
-        </MarqueeSelection>
-      </ScrollablePane>
+    <div>
+      <TextField
+        className={classNames.filter}
+        label="Filter by name:"
+        // eslint-disable-next-line react/jsx-no-bind
+        onChange={onFilterChange}
+      />
+      <h1 className={classNames.header}>Item list</h1>
```

Comment:
While you're modifying this, might as well change this to something with proper nesting (probably h3 or h4) 🤦‍♀️

Reply:
On second thought probably just remove the header; there's really no good reason for it to be here.